include/code_800F958.h

```diff
@@ -0,0 +1,42 @@
+#ifndef GUARD_CODE_800F958_H
+#define GUARD_CODE_800F958_H
+
+#include "structs/str_dungeon.h"
+
+// size: 0x8
+typedef struct DungeonPokemonStatusSprite
+{
+    /* 0x0 */ u32 status;
+    /* 0x4 */ u32 frame;
+} DungeonPokemonStatusSprite;
+
+// size: 0x40
+typedef struct DungeonPokemonSprite
+{
+    /* 0x0 */ u8 exists;
+    /* 0x4 */ u32 id;
+    /* 0x8 */ u16 species; // Or sprite ID?
+    /* 0xA */ u16 unkA;
+    /* 0xC */ u32 status;
+    /* 0x10 */ u8 visible;
+    /* 0x11 */ u8 unk11;
+    /* 0x14 */ Position pos;
+    /* 0x18 */ Position statusOffsets[2];
+    /* 0x20 */ u32 unk20;
+    /* 0x24 */ u32 unk24;
+    /* 0x28 */ DungeonPokemonStatusSprite statusSprites[2];
+    /* 0x38 */ u8 unk38;
+    /* 0x3C */ u16 unk3C;
+    /* 0x3E */ u16 unk3E;
+} DungeonPokemonSprite;
+
+// size: 0x584
+typedef struct DungeonPokemonSprites
+{
+    /* 0x0 */ u32 frame;
+    /* 0x4 */ struct DungeonPokemonSprite sprites[22];
```

Comment:
Not really needed but the `struct` keyword was hiding an issue when I was decomping the relevant functions. It said `struct DungeonPokemonSprites` (with an s) which means the struct was referring to itself without generating a compiler error. It's why I prefer removing the struct keyword since we already typedef'd above. In my branch I put it as `DungeonPokemonSprite`

Reply:
It might have been left there in the decomp.me scratch. I'll remove `struct` and make use of the typedef.